backend/src/main/java/io/debezium/configserver/rest/client/JolokiaClient.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.configserver.rest.client;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.management.MalformedObjectNameException;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jolokia.client.BasicAuthenticator;
+import org.jolokia.client.J4pClient;
+import org.jolokia.client.exception.J4pException;
+import org.jolokia.client.request.J4pReadRequest;
+import org.jolokia.client.request.J4pReadResponse;
+
+@ApplicationScoped
+public class JolokiaClient {
+    public static final int DEFAULT_JOLOKIA_PORT = 8778;
+
+    public static final String PROPERTY_JOLOKIA_CLIENT_URI = "jolokia.client.uri";
```

Comment:
```
Caused by: jakarta.enterprise.inject.spi.DeploymentException: io.quarkus.runtime.configuration.ConfigurationException: Failed to load config value of type class java.lang.String for: jolokia.client.uri
```
Is causing the test failure. I think you need to remove this one and all usages.

Reply:
So I suggest we try and be flexible in this regard.

I see in `ConnectoResource` line 549, we have this logic:
```
String.format("http://%s:%d/jolokia", kafkaConnectURI.getHost(), JolokiaClient.DEFAULT_JOLOKIA_PORT);
```

I think this is a great way to handle the default situation where the user may be using Jolokia bundled with our container images; however, this may prove problematic for other installations where the agent may have been installed specifying an `agentContext` other than the default of `jolokia` or using a non-default port.

So I think there is merit in allowing users to override this URI in the application properties if they have added the Jolokia client in a way that deviates from using the default agent configuration values.  In order to support this, the change is quite simple to make the code that Anisha has already in place work for both.

First, you can make a `@ConfigProperty` binding not be required by wrapping the variable as an `Optional`, and this will allow Quarkus to safely start without failure:
```
    @ConfigProperty(name = PROPERTY_JOLOKIA_CLIENT_URI)
    private Optional<String> configJolokiaUrl;
```
Then I'd suggest modifying line 39 of the `JolokiaClient` to:
```
        final J4pClient client = getJolokiaClient(configJolokiaUrl.orElse(jolokiaUrl));
```
Running the test suite with this modification works as expected regardless of whether the Jolokia endpoint is constructed from the hardcoded template in the resource or by being specified in application properties.

Reply:
@Naros  How is this going to work with users operating multiple Connect clusters from a single UI?

Reply:
> I think this is a great way to handle the default situation where the user may be using Jolokia bundled with our container images;

I think in this phase it is okay to say this is the only supported way to run UI, using our images or configure Jolokia in a compatible way. When we move to the final solution  based on the Debezium Connect REST Extension this all will go away anyway, right?

Reply:
Would it be reasonable @rk3rn3r and @ani-sha to treat the `jolokia.client.uri` property in the same way we do the Kafka Connect URI property, that if multiple values are provided for Kafka Connect that we expect a similar comma-separated list of URIs for Jolokia?

Reply:
It would, but I think the effort is to high considering that we are going to move to another solution when migrating to the Debezium Connect REST Extension. Let's just rewrite the Kafka Connect URIs that come from the `{{cluster}}` URL parameter. Imo it is a good enough assumption to enable the feature and finally have it out of the door (with those standard config limitations). We can do things properly when we move it to the REST extension.